pkg/console/subresource/managedserviceproxyresolver/thanos-querier.go

```diff
@@ -0,0 +1,39 @@
+package managedproxyserviceresolver
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	// openshift
+
+	"github.com/openshift/console-operator/pkg/api"
+	"github.com/openshift/console-operator/pkg/console/subresource/util"
+	proxyv1alpha1 "open-cluster-management.io/cluster-proxy/pkg/apis/proxy/v1alpha1"
+)
+
+func DefaultThanosQuerierProxyServiceResolver() *proxyv1alpha1.ManagedProxyServiceResolver {
+	return &proxyv1alpha1.ManagedProxyServiceResolver{
```

Comment:
so this looks like a static manifest, lets put it into the bindata

Reply:
The problem is we don't have a 'ReadOrDie' func for ManagedProxyServiceResolver resources.